spec/lexers/awk_spec.rb

```diff
@@ -2,7 +2,8 @@
 # frozen_string_literal: true
 
 describe Rouge::Lexers::Awk do
-  let(:subject) { Rouge::Lexers::Awk.new }
+  let(:described_class) { Rouge::Lexers::Awk }
```

Comment:
Same as above.

Reply:
Yes. `Rouge::Lexers::Awk` implements the `:detect?` singleton_method. So it is being used to test for false negative response from the new singleton_method `:detectable?`